### DISCUSSION TITLE
How to implement Preview Pull Request in Bitbucket Pipelines CI/CD?

### DISCUSSION THREAD

**Discussion by vicasas** [vercel/vercel#7752] [Help] [answered]
I have a repository on bitbucket which is deployed to firebase and we adhere to the flow of always doing a pull request to merge `master`. Based on this scenario we need to implement pull request previews so that the team can review the new features.

Based on the above we want to use Vercel to implement previews, but we only want this feature no more no less. The idea is that using the CD stage of our `bitbucket-pipelines.yml` file we can configure this and get a preview URL for each pull request.

I couldn't find anything in the documentation about this scenario. Is it possible this way using Vercel?

The closest thing I've seen so far is on the Firebase side: https://firebase.google.com/docs/hosting/test-preview-deploy

**Comment by anishcorratech** [comment, marked as answer]
I am new to vercel myself, But I got this page from vercel for bitbucket pipelines:

https://vercel.com/guides/how-can-i-use-bitbucket-pipelines-with-vercel

**Reply by vicasas** [reply]
wow! I haven't tried it, but I think this leads to what I was looking for. Thank you!
